src/commands/until_front_of_line.yml

```diff
@@ -104,15 +109,15 @@ steps:
         fetch_filtered_active_builds(){
           if [ "<<parameters.consider-branch>>" != "true" ];then
             echo "Orb parameter 'consider-branch' is false, will block previous builds on any branch." 
-            jobs_api_url_template="https://circleci.com/api/v1.1/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}?filter=running"
+            jobs_api_url_template="<< parameters.circleci-hostname >>/api/v1.1/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}?filter=running"
```

Comment:
Declare as BASH variable once near beginning (like https://github.com/eddiewebb/circleci-queue/blob/main/src/commands/until_front_of_line.yml#L47) and then use env_var thorugh script.

This was authored before support for `include` but I want to move that way ans so avoid as much inline parameters as possible for now.

`CIRCLECI_BASEURL="http://<< parameters.circleci-hostname >>"`
And that would be a great line to assert in the tests